.circleci/config.yml

```
@@ -43,15 +48,24 @@ commands:
             uv venv
             source .venv/bin/activate
             uv pip install .
-            uv pip install -r ./test_requirements/requirements_<<parameters.py>>_optional.txt
+            uv pip install -r ./test_requirements/requirements_optional.txt
             cd js
             npm ci
             npm run build
-      - run:
-          name: Install plotly-geo
-          command: |
-            source .venv/bin/activate
-            uv pip install plotly-geo
+      
+      - when:
+          condition: 
+            not:
+              equal:
+                - <<parameters.pandas_version>>
+                - ""
+          steps:
+            - run:
+                name: Install pandas
+                command: |
+                  source .venv/bin/activate
+                  pip install pandas==<<parameters.pandas_version>>
```

@marthacryan I think this should be 
```suggestion
                  uv pip install pandas==<<parameters.pandas_version>>
```

---

That actually caused issues for some reason.. I had that originally and it threw a weird error.

---

Otherwise `uv pip install pip && python -m pip install` should work